src/main/java/uk/gov/hmcts/ccd/CachingConfiguration.java

```diff
@@ -37,11 +81,43 @@ private void configCaches(int definitionCacheTTL, Config config) {
         config.addMapConfig(newMapConfig("userRolesCache", definitionCacheTTL));
     }
 
+    private void warmUp(final HazelcastInstance hazelcastInstance) {
+        IMap<String, CaseType> caseTypeDefinitionsCache = hazelcastInstance.getMap("caseTypeDefinitionsCache");
+        List<CaseType> allCaseTypes = getAllCaseTypes();
+        allCaseTypes.stream().forEach(caseType -> {
+            caseTypeDefinitionsCache.put(caseType.getId(), caseType);
```

Comment:
Can we introduce a CachePreWarmingService Interface with on implementation for the moment DefaultCachePreWarmingServiceImpl please. So we have all the logic
for the pre warming nicely encapsulated

Reply:
Yes I will introduce DefaultCacheWarmUpService as that name sounds better to me.